### ISSUE TITLE
Allow us to customize the order of spell lore

### ISSUE THREAD

**Issue by grisstyl** [elBukkit/MagicPlugin#590]
I've been combing through the message defaults, `default` spell properties, and asked the Discord about changing the order of spell lore entries to no avail. I know this is possible for wands through the `lore` property.

If this is possible I can't find the default format which is necessary so that I don't miss any placeholders.

**Comment by NathanWolf** [comment]
Yeah I don't think you can do this with spells like you can with wands. Would be nice to have, though.
